media/js/mozorg/manifesto.js

```diff
@@ -148,6 +16,7 @@ $(function() {
         // Check if the official Twitter widget is activated. If so, we don't
         // have to open a popup window ourselves as it will be opened by the
         // widget code.
+        // https://dev.twitter.com/docs/intents
```

Comment:
It looks like this link has changed to https://dev.twitter.com/web/intents

As for `window.__twitterIntentHandler`, I think we should be ok to remove this. We wouldn't embed a twitter widget in pages on mozorg since it violated our websites privacy policy. Not entirely sure why this was here to begin with, as I see no history of the widget ever being on the page.